libs/template/src/lib/render-strategies/strategies/global.strategy.ts

```diff
@@ -13,14 +13,20 @@ export function getGlobalStrategies(
 }
 
 /**
+ *
+ * @description
+ *
  * Global Strategies
  *
- * - ɵMD - `ɵmarkDirty`
- * - C - `Component`
+ * This strategy leverages Angular's internal `ɵmarkDirty` render method.
+ * This means it acts identical to `ChangeDetectorRef#markForCheck` but works also zone-less.
```

Comment:
```suggestion
 * This means it acts identical to `ChangeDetectorRef#markForCheck` but works also zone-less.
 * see: https://github.com/angular/angular/blob/930eeaf177a4c277f437f42314605ff8dc56fc82/packages/core/src/render3/view_ref.ts#L128
 * see: https://github.com/angular/angular/blob/930eeaf177a4c277f437f42314605ff8dc56fc82/packages/core/src/render3/instructions/change_detection.ts#L36
```

Reply:
maybe we should add links to the angular repository?

also: `markDirty` calls `scheduleTick(rootView[CONTEXT] as RootContext, RootContextFlags.DetectChanges);` which `markForCheck` doesn't. That is why `markDirty` works in zone-less environments

Reply:
Totally agree! We should add that!

What about: 
> This strategy leverages Angular's internal [`ɵmarkDirty`](https://github.com/angular/angular/blob/930eeaf177a4c277f437f42314605ff8dc56fc82/packages/core/src/render3/instructions/change_detection.ts#L36) render method.
> It acts identical to [`ChangeDetectorRef#markForCheck`](https://github.com/angular/angular/blob/930eeaf177a4c277f437f42314605ff8dc56fc82/packages/core/src/render3/view_ref.ts#L128) but works also zone-less.
> `markDirty` in comparison to `markForCheck` also calls [`scheduleTick`](https://github.com/angular/angular/blob/930eeaf177a4c277f437f42314605ff8dc56fc82/packages/core/src/render3/instructions/shared.ts#L1863) which is the reason why it also works in zone-less environments.

Reply:
Done